### ISSUE TITLE
Game chat is not very adapted to chat

### ISSUE THREAD

**Issue by Marsux** [bnordli/rftg#12]
Currently, the chat message in the game are displayed in the same widget as the journal log. While this is good to comment the game, it makes difficult to have a conversation through the chat, because messages are mixed with sometimes large log entries.

Do you think it would be possible to separate the two type of messages (chat and log entries?) or maybe replicate the chat messages both in the log window and in an additional chat window?

**Comment by dsnvwlmnt** [comment]
While streaming i noticed that chat text tends to just bleed into the rest. I think at the least if you leave chat inside the log, you could change the look and color of chat messages. Black on white like most of the rest just isn't very visible. You could do reverse coloring, i.e. white text on solid black background or something. But yeah, better yet would be separate.

**Comment by bnordli** [comment]
I agree that the mixing of messages is confusing, and that a separate chat window would work better. No time to do this before 0.9.5, though.

**Comment by dsnvwlmnt** [comment]
Is it possible to change chat text to red? Keeping the bold on the player names.

**Comment by bnordli** [comment]
Suggestion: #009900.
![chat_color](https://cloud.githubusercontent.com/assets/5574761/20425714/0d0263b2-ad7c-11e6-977d-7d0cc29eb4ed.png)

**Comment by Marsux** [comment]
Didn't have time to comment before: part of the issue is that you have to scroll in order to retrace the conversation.

A suggestion of solution: add a way to toggle the display of log entries. Under the hood, there would be two display widgets, one that gets log and chat entries, the other one only chat entries. Only one of the widget is displayed in the usual location.

**Comment by dsnvwlmnt** [comment]
Conversation continued in [Issue#17](https://github.com/bnordli/rftg/issues/17).

**Comment by bnordli** [comment]
Yes, this clearly needs more thought. I am dropping the color change, too, waiting for a better solution.

**Comment by dsnvwlmnt** [comment]
I'm assuming this issue won't be solved in the XI release, so I would prefer if the color change remained.
